kuma/wiki/tests/test_templates.py

```diff
@@ -1329,3 +1329,110 @@ def _translation_data():
         'content': 'loremo ipsumo doloro sito ameto',
         'toc_depth': Revision.TOC_DEPTH_H4,
     }
+
+
+def test_zone_styles(client, doc_hierarchy_with_zones, root_doc):
```

Comment:
Covers all the test cases. I might have tried to parameterize it, but it is clear enough as one after the other.